### ISSUE TITLE
Problem on guides - Getting started with Engines

### ISSUE THREAD

**Issue by superbull** [rails/rails#19453]
https://github.com/rails/rails/blob/master/guides/source/engines.md

on line 411: 
```
<%= stylesheet_link_tag "scaffold" %>
```

When start the server, and pointing to `http://localhost:3000/blorgh/articles`. An exception is thrown:
![screen shot 2015-03-22 at 4 43 23 pm](https://cloud.githubusercontent.com/assets/455630/6768630/a28ecfb8-d0b2-11e4-8efd-7b2250aa74db.png)

Add one line to the lib/blorgh/engine.rb solved this issue:
```
module Blorgh
  class Engine < ::Rails::Engine
    isolate_namespace Blorgh
    
    # add line
    self.config.assets.precompile += %w( scaffold.css )
  end
end
```

Is it the right way to solve this?

**Comment by kaspth** [comment]
Please use StackOverflow or the rubyonrails talk mailing list for questions like this one :heart:

That being said the error message already tells you how to solve the problem.

**Comment by matthewd** [comment]
@kaspth this is a report of an issue in a guide.

**Comment by kaspth** [comment]
Wups, sorry! :)

**Comment by wazery** [comment]
Can add a label for `attached PR`, right?

**Comment by zzak** [comment]
Fixed by da5fe3be1f31ec6ae695df0265506e2b2f954663
